Library/Formula/apkpatch.rb

```diff
@@ -0,0 +1,19 @@
+class Apkpatch < Formula
+  desc "Tool for build/merge Android Patch file(.apatch)"
+  homepage "https://github.com/alibaba/AndFix"
+  url "https://raw.githubusercontent.com/alibaba/AndFix/master/tools/apkpatch-1.0.3.zip"
+  sha256 "0f94cd2a675af89ff3990f4b6490dd70877ad4eb109f0cdbefece1f9a73e4512"
+
```

Comment:
Just checked the source and it appears `akpatch` needs Java to work. Please add `depends_on :java` here. It should look like this:

```rb
  # ...
  sha256 "0f94cd2a675af89ff3990f4b6490dd70877ad4eb109f0cdbefece1f9a73e4512"

  depends_on :java

  def install
    # ...
```